tests/manager/core/test_metadata.py

```diff
@@ -49,675 +46,613 @@ def test_constructor(self):
         assert 0.5 == data.weight
 
 
-class CSVFilesFixture(FilesFixture):
-    """A fixture providing access to CSV files."""
+class TestContributorData:
+    def test__init__(self):
+        # Roles defaults to AUTHOR
+        assert ContributorData().roles == [Contributor.Role.AUTHOR]
 
-    def __init__(self):
-        super().__init__("csv")
+        # If roles is a string, it is converted into a list
+        assert ContributorData(roles="foo").roles == ["foo"]
 
+        # if roles is a sequence (tuple, list, etc), it is copied to a list
+        assert ContributorData(roles=("x", "y")).roles == ["x", "y"]
+        assert ContributorData(roles=["x", "y"]).roles == ["x", "y"]
 
-@pytest.fixture()
-def csv_files_fixture() -> CSVFilesFixture:
-    """A fixture providing access to CSV files."""
-    return CSVFilesFixture()
+    def test_from_contribution(self, db: DatabaseTransactionFixture):
+        # Makes sure ContributorData.from_contribution copies all the fields over.
 
+        # make author with that name, add author to list and pass to edition
+        contributors = ["PrimaryAuthor"]
+        edition, pool = db.edition(with_license_pool=True, authors=contributors)
 
-class TestMetadataImporter:
-    def test_parse(self, csv_files_fixture: CSVFilesFixture):
```

Comment:
The diff here is confusing, but `test_parse` is the only test that we removed here. The other tests in `TestMetadataImporter` were still relevant, so I moved them down in the file into `TestMetadata` since that is what they are actually testing. This made for a kind of messy diff, but it preserves almost all the tests.